css/style.css

```diff
@@ -502,3 +528,349 @@ form {
+    background: url("../images/Group\ desktop.svg");
+    background-repeat: no-repeat;
+    background-position: 1098px 0;
+  }
+
+  .about-me-text {
+    width: 100%;
+    height: 345px;
+    background: url("../images/Group\ desk.svg");
+    background-repeat: no-repeat;
+    background-position: 1px;
+  }
+
+  .about-me-text p {
+    font-weight: 400;
+    font-size: 20px;
+    color: #42526e;
+    margin-top: 10px;
+    width: 920px;
+    height: 96px;
+    padding-left: 350px;
+    padding-top: 20px;
+    text-align: center;
+  }
+
+  .get-resume {
+    transition: 0.5s;
+  }
+
+  .about-me .get-resume {
+    margin-top: 60px;
+  }
+
+  .my-knowledge {
+    width: 1171px;
+    margin: auto;
+    display: grid;
+    gap: 2px;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .my-knowledge .languages,
+  .my-knowledge .frameworks,
+  .my-knowledge .skills {
+    width: 369px;
+    height: 375px;
+  }
+
+  .contact-form .form-flex {
+    margin: 0 80px;
+    display: grid;
+    width: 1171px;
+    height: 490px;
+    gap: 10px;
+    grid-template-columns: 1fr 2fr;
+  }
+
+  .contact-form .contact-me {
+    width: 463px;
+    height: 253px;
+  }
+
+  .contact-me-content {
+    height: 200px;
+    width: 463px;
+    padding: 0;
+    padding-right: 25px;
+    margin: 0;
+  }
+
+  .contact-me-content h3 {
+    color: #172b4d;
+    width: 463px;
+    font-weight: 700;
+    font-size: 30px;
+  }
+
+  form {
+    position: relative;
+  }
+
+  .form-group {
+    margin-right: 0;
+  }
+
+  .email,
+  .contact-shape {
+    display: none;
+  }
+
+  .email {
+    height: 200px;
+    width: 500px;
+  }
+
+  hr {
+    background: #dfe1e6;
+    width: 1440px;
+    height: 1px;
+    display: inline-block;
+    margin-top: 100px;
+  }
+
+  .contact-container {
+    width: 100%;
+    height: 100vh;
+    background-image: url('../images/Frame.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: bottom 2px right -25px;
+    margin-bottom: 100px;
+  }
+
+  .contact-container .social-icons {
+    margin-left: 580px;
+  }
+
+  .contact-illus-1 {
+    height: 160px;
+    width: 110px;
+    margin-top: 0;
+    padding-top: 0;
+
+    /* background-image: url('../images/contact\ illus-1.svg'); */
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: 5px 10px;
+  }
+
+  .contact-illus-2 {
+    height: 50px;
+    width: 10px;
+    margin-left: 13px;
+  }
+
+  .form-group #fullname {
+    display: none;
+  }
+
+  .first-last-name {
+    display: inline-block;
+  }
+
+  form .first-last-name {
+    display: grid;
+    gap: 10px;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .get-in-touch {
+    margin-left: 15px;
+  }
+
+  .icon-resize {
+    position: absolute;
+    top: 47%;
+    right: 7px;
+  }
+
+  .works-button {
+    transition: 0.5s;
+  }
+
+  .works-button:hover {
+    background-color: green;
+  }
+
+  .get-resume:hover {
+    background-color: rgb(141, 12, 12);
+  }
```

Comment:
- [ ] Kindly add the right color for the hover effect,  the button has the same color when  enabled or  hovered please check the Figma design, good job adding the right button  effect when the button is clicked, please do this for all buttons as this was requested by [previous reviewer](https://github.com/yayner2002/Portfolio-setup-and-mobile-version-skeleton/pull/17#discussion_r834710649) 👍🏿 

![Screenshot_36](https://user-images.githubusercontent.com/69638013/160090509-85d7a385-cf1d-4c1b-b4b4-145c2de13e97.png)